src/containers/MainPage/CreateRoomModal/index.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+
+import injectReducer from 'utils/injectReducer';
+import injectSaga from 'utils/injectSaga';
+
+import { Modal, Button, Input, message, Icon } from 'antd';
+
+import { setRoomHost } from 'redux/room/actions';
+import { REDUCER_KEY } from './constants';
+import { reset, createRoom } from './actions';
```

Comment:
How about remove duplicated action? change to below
import { createRoomActions } from './actions';